components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -86,19 +86,31 @@ export function WorkspaceEntry({ desc, model, isAdmin, stopWorkspace }: Props) {
                 <WorkspaceStatusIndicator instance={desc?.latestInstance} />
             </div>
             <div className="flex flex-col w-3/12">
-                <a href={startUrl.toString()}><div className="font-medium text-gray-800 dark:text-gray-100 truncate hover:text-blue-600 dark:hover:text-blue-400">{ws.id}</div></a>
-                <a href={project ? 'https://' + project : undefined}><div className="text-sm overflow-ellipsis truncate text-gray-400 hover:text-blue-600 dark:hover:text-blue-400">{project || 'Unknown'}</div></a>
+                <Tooltip content={ws.id}>
+                    <a href={startUrl.toString()}>
+                        <div className="font-medium text-gray-800 dark:text-gray-100 overflow-ellipsis truncate w-full hover:text-blue-600 dark:hover:text-blue-400">{ws.id}</div>
+                    </a>
+                </Tooltip>
+                <Tooltip content={project || 'Unknown'}>
+                    <a href={project ? 'https://' + project : undefined}>
+                        <div className="text-sm overflow-ellipsis truncate w-full text-gray-400 hover:text-blue-600 dark:hover:text-blue-400">{project || 'Unknown'}</div>
+                    </a>
+                </Tooltip>
             </div>
-            <div className="flex w-4/12 truncate overflow-ellipsis">
-                <div className="flex flex-col">
-                    <div className="text-gray-500 overflow-ellipsis truncate">{ws.description}</div>
+            <div className="flex flex-col w-4/12">
+                <Tooltip content={ws.description}>
```

Comment:
**issue**: Adding the tooltip on all element can a) make the interface feel slightly overwelmed, b) feel redandunt on some elements like the workspace name, and introduce some visual imbalance (see screenshots).

| Context | Changes | 
|-|-|
| <img width="404" alt="Screenshot 2021-06-10 at 2 22 19 AM" src="https://user-images.githubusercontent.com/120486/121442020-7679c780-c993-11eb-8026-b53a3f2db7e5.png"> | <img width="248" alt="Screenshot 2021-06-10 at 2 22 10 AM" src="https://user-images.githubusercontent.com/120486/121442025-77aaf480-c993-11eb-9c15-4af6074a003f.png"> |

Reply:
Yes, absolutely. Since the truncation is fixed in #4454 already, I'll close this one.

Reply:
Thanks @corneliusludmann! 🏓